### DISCUSSION TITLE
Not about Decompose

### DISCUSSION THREAD

**Discussion by lamtech58** [arkivanov/Decompose#621] [Q&A]
I asked this question at many places but never got a clear answer, here it is : it's about the new Google play store policy 20 testers before publishing an app for developer new account. My question is who are those testers? Can they be anyone who has an android and can access Google play store? Or it's only about developers who have developer account?

Sorry it's not Decompose related question

**Comment by arkivanov** [comment]
I think the policy is described [here](https://support.google.com/googleplay/android-developer/answer/14151465). Seems like testers don't need a developer account.

**Reply by lamtech58** [reply]
Alright thanks
